app/routes.py

```diff
@@ -1,2 +1,218 @@
+            video.resp_json()
+        ), 200
+
+    elif request.method == "PUT":
+        about_video = request.get_json()
+        if "title" not in about_video or "release_date" not in about_video or "total_inventory" not in about_video:
+            return make_response({"details": "invalid data"}, 400)
+
+        video.title = about_video["title"]
+        video.release_date = about_video["release_date"]
+        video.total_inventory = about_video["total_inventory"]
+        
+        db.session.commit()
+        return jsonify(
+            video.resp_json()
+        ), 200
+       
+    elif request.method == "DELETE":
+
+        db.session.delete(video)
+        db.session.commit()
+
+        return { "id": video.video_id}, 200
+    
+                            ######## CRUD RENTAL ########
+
+@rentals_bp.route("/check-out", methods = ["POST"])
+def video_check_out():
+    request_body = request.get_json()
+
+    if "video_id" not in request_body or "customer_id" not in request_body:
+        return make_response({"details":"invalid info"}, 400)
+
+    if not isinstance(request_body["video_id"], int) or not isinstance(request_body["customer_id"], int):
+        return make_response({"details":"invalid info"}, 400)
+
+    video = Video.query.get_or_404(request_body["video_id"])
+    customer = Customer.query.get_or_404(request_body["customer_id"])
+
+    
+    if video.available_inventory == None:
+        return make_response({"details":"inventory unavailable"}, 400)
+    video.available_inventory -=1
+    customer.videos_checked_out_count +=1
+    
+    rental = Rental (
+        customer_id = request_body["customer_id"],
+        video_id = request_body["video_id"]
+        )
+    db.session.add(video)
+    db.session.add(customer)
+    db.session.add(rental)      
+    db.session.commit()
+    return jsonify(
+        rental.rental_ops()
+    ), 200
+
+@rentals_bp.route("/check-in", methods = ["POST"])
+def video_check_in():
+    request_body = request.get_json()
+
+    if "video_id" not in request_body or "customer_id" not in request_body:
+        return make_response({"details":"invalid info"}, 400)
+
+    if not isinstance(request_body["video_id"], int) or not isinstance(request_body["customer_id"], int):
+        return make_response({"details":"invalid info"}, 400)
+
+    video = Video.query.get_or_404(request_body["video_id"])
+    customer = Customer.query.get_or_404(request_body["customer_id"])
+    if video.available_inventory == None:
+        video.available_inventory = 1
+    else:
+        video.available_inventory +=1
+    customer.videos_checked_out_count -=1
+    
+    rental = Rental (
+        customer_id = request_body["customer_id"],
+        video_id = request_body["video_id"]
+        )
+    json_body = rental.rental_ops()
+    db.session.add(video)
+    db.session.add(customer)  
+    # db.session.delete(rental) 
+    db.session.commit()
+    
+    return jsonify(
+        json_body
+    ), 200
```

Comment:
Great job setting up the routes Juliana! Optionally, you could separate the routes for customers, videos, and rentals into separate files under a folder called 'routes'. That's a typical project structure you'll see in wild.